src/main/java/com/google/sps/models/App.java

```diff
@@ -14,4 +14,14 @@
         this.rating = rating;
         this.price = price;
     }
+
+    App setRated(String rated) {
+        this.rated = rated;
+        return this;
+    }
+
+    App setAppDescription(String appDescription) {
+        this.appDescription = appDescription;
+        return this;
+    }
```

Comment:
Did you have problems with the Builder pattern?
https://dzone.com/articles/design-patterns-the-builder-pattern

```
public App {

    public static class Builder {
       ...
   }
   ...

}
```

It usually looks this when being consumed
```
App.Builder.setAppDescription("bleh").build()
```

Instead of
```
App.setAppDescription("bleh")
```

Reply:
i changed it to build in the new commit but, it doesn't seem to work when i do App.Builder.setName()....build()
but it works when i do App().setName(" ")....build()